heft-plugins/heft-jest-plugin/package.json

```
@@ -11,11 +11,11 @@
   "types": "dist/heft-jest-plugin.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "heft test --clean",
+    "build": "heft build --clean && jest --rootDir=./ --config=./config/jest.json",
```

Can we keep the jest config following our naming convention?

```suggestion
    "build": "heft build --clean && jest --rootDir=./ --config=./config/jest.config.json",
```

Also why won't `heft test` work?